### PR TITLE
Fork Concourse windows image build pipeline for MBR

### PR DESCRIPTION
**Pull request by jjerger**
Adds MBR builds as a separate pipeline to be run ad-hoc

### CODE REVIEW

**Comment by jjerger** [comment]
Question for Brian and Zach: Does it make sense to define the pipeline assuming that only the "testing" env will ever be passed in? I had removed alot of the branching logic from the original pipeline that stemmed from there being 4 publishing environments.

File: concourse/pipelines/windows-image-build-mbr.jsonnet [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+          ImgPublishJob(image, env, 'windows', 'windows-bios')
+          for image in windows_server_images
+          for env in envs
+        ],
+
+  groups: [
+    ImgGroup('windows-2012-mbr', windows_2012_images, envs),
+    ImgGroup('windows-2016-mbr', windows_2016_images, envs),
+    ImgGroup('windows-2019-mbr', windows_2019_images, envs),
+    ImgGroup('windows-2022-mbr', windows_2022_images, envs),
```
Comment (zmarano) [inline concourse/pipelines/windows-image-build-mbr.jsonnet:344]: I don't believe we need to build 2022 as an MBR image- it never existed as MBR. WDYT?

Comment (bkatyl) [reply concourse/pipelines/windows-image-build-mbr.jsonnet:344]: We need it because customers can import Windows Server 2022 MBR.

**Comment by bkatyl** [comment]
As we talked about in chat, lets remove the core images since these will only be for boot tests and we don't need or want to encourage double testing.

File: concourse/pipelines/windows-image-build-mbr.jsonnet (hunk 1 above)
Comment (zmarano) [reply concourse/pipelines/windows-image-build-mbr.jsonnet:344]: Good point, or in place upgrades. Ok.

**Review by bkatyl — APPROVED** [review]

**Pull request merged**